worksheet.rb

```diff
@@ -23,12 +39,159 @@
 # into this data structure, such as "DR0004"
 # and "3rd Feb 2016" and "RD0022"
 
+drivers = [
```

Comment:
Great work formatting your data to make it very readable. Your data is a bit off from the data in the csv, but your calculations are correct considering this mis-entry.